cfgov/jinja2/v1/sublanding-page-1/index.html

```diff
@@ -33,22 +19,46 @@
         - Sidebar with latest activities
 #}
 
+{% block pre_content %}
+    {% set settings = {
+        'heading':          'Sample Header(H3)',
+        'hide_header_slug': true,
+        'image': {
+            'url':  '/static/img/clipboard-round-300x300.png',
+            'round' : true
+        },
+        'intro':   {
+            'heading': 'Sample Header(H4)',
+            'body':    lipsumText,
+            'icon':    'speech-bubble'
+        },
+        'related_posts': {
+            'view_more': {
+                'href': '/',
+                'text': 'View More Items'
+            },
+            'slug': 'four-years-working-for-you'
+        }
+       }
+    %}
+    {% call sidebar_breakout.render(settings) %}
+        {{ text_introduction.render({
+            'heading': 'Text Introduction',
```

Comment:
I don't think we have a rule on aligning values, but you've done it above so might as well here too.